eisenhower-matrix/src/main/webapp/script.js

```diff
@@ -119,8 +89,9 @@ async function getTasks() {
         let deleteButton = document.createElement("button");
         deleteButton.innerText = "X";
         deleteButton.style.color = "red";
-        deleteButton.addEventListener("click", function() {
+        deleteButton.addEventListener("click", function(e) {
             deleteComment(task.id);
+            e.preventDefault();
             curr_task.style.display = "none";
```

Comment:
Not sure, but probably you can just do `curr_task.style.display = none` ? If so I think that's the way to go